ui/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const zIndexLanguageSwitcher = 100
+export const zIndexModal = 500
+export const zIndexLoadingOverlay = 1000
```

Comment:
Maybe we should rename this file? zIndexConstants?

Reply:
sure! 👍